docs/user-guide/troubleshooting.md

```diff
@@ -0,0 +1,23 @@
+# Troubleshooting
```

Comment:
Nice addition. How will people find this page? Do we need to link it from the main page?

Reply:
Good catch, it was actually missing.
In the `mkdocs.yaml` I added `- user-guide/troubleshooting.md` to the `User guide` section to make it work.